CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+cmake_minimum_required(VERSION 3.12.0)
+
+project(pure-maps
+	VERSION 2.6.5
+	DESCRIPTION "Maps and navigation")
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+include(FeatureSummary)
+include(GNUInstallDirs)
+include(CMakePrintHelpers)
+
+option(USE_BUNDLED_GPXPY "Use a bundled version of GPXPY rather than a system-wide version" ON)
+option(FLAVOR "Platform to build support for. Supported platforms: kirigami, silica, qtcontrols, uuitk" OFF)
+set(PROFILE "Online" CACHE STRING "Default profile. Supported profiles: Online, Mixed, Offline")
+option(RUN_FROM_SOURCE "Run from source, this is mainly intended for easy development" OFF)
+
```

Comment:
For readability, I would add S2 options here as well and set them to OFF. Although, I can add it later as well

Reply:
Not possible. Options can only be ON or OFF (or TRUE and FALSE). The S2 options however are strings, so would need to be set with `set`. That however means I need to give it an explicit value, which we don't want here.

Reply:
We can probably set them to empty strings then and apply only if the string has some length